Fix OC promotional inbox note CTA (#4512)
woocommerce/woocommerce-gateway-stripe: changelog.txt

*** Changelog ***

= 9.8.0 - xxxx-xx-xx =

> * Fix - Fixes the CTA button on the Optimized Checkout promotional inbox note to correctly link to the Stripe settings page

Reviewer: Non-blocking: I think we can simplify the changelog
```suggestion
* Fix - Update the Optimized Checkout promotional inbox note to link to the relevant section in the Stripe settings page
```
